app/aggregates/deciding/overall_result_calculator.rb

```diff
@@ -0,0 +1,60 @@
+module Deciding
+  class OverallResultCalculator
+    def initialize(decision)
+      @decision = decision
+    end
+
```

Comment:
The overall result is now displayed to the user in the format `funding_decision - qualification`, e.g., "Granted - with contribution" or "Refused - failed IoJ".  

In the datastore, these values are stored as types,  such as `granted_with_contribution` and `refused_failed_ioj`. 

This class determines the `OverallResult` type.